src/test/kotlin/com/bardsoftware/backend/fleet/rmarkdown/PandocTest.kt

```diff
@@ -0,0 +1,36 @@
+/**
+    Copyright 2018 BarD Software s.r.o
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Author: Mikhail Shavkunov (@shavkunov)
+ */
+package com.bardsoftware.backend.fleet.rmarkdown
+
+import org.junit.Test
+import java.nio.file.Paths
+import kotlin.test.assertEquals
+
+class PandocTest {
```

Comment:
So let's add a test which tests not only substitution, but also that command (`cp`) is executed and result appears on disk?

Reply:
I've removed `substituteTest', because this was testing `Substitutor` logic, not ours and replaced with test, that, in my opinion, tests the production logic of compile markdown to tex.